projects/blinking_led/src/main.c

```diff
@@ -0,0 +1,34 @@
+#include "interrupt.h"   // apparently soft timers needs this
```

Comment:
These comments are okay for your learning and for this homework.
Normally we wouldn't put these comments since they are sort of self explanatory, so for actual projects I would probably not put these sorts of comments in.

Reply:
alright, thank you very much for the help